### PR TITLE
fix: toString method can now handle 0 as number

### PR DESCRIPTION
**Pull request by mok-liee**
hi, i think in some situations it is necessary to have a numeric 0 in the condition.

### CODE REVIEW

**Comment by mok-liee** [comment]
by the way, sorry. i forgot to create a feature branch.

File: src/comparators.ts [hunk 1]
```diff
@@ -7,14 +7,12 @@ export abstract class Comparator implements Item {
   constructor(protected selector: Function | string, protected comparatorKeyWord: string, protected value?: string | number) {}
 
   toString(): string {
-    if (this.value) {
-      if (typeof this.value === 'number') {
-        return `${this.selector} ${this.comparatorKeyWord} ${this.value}`;
-      } else {
-        return `${this.selector} ${this.comparatorKeyWord} '${this.value}'`;
-      }
+    if (typeof this.value === 'undefined' || this.value === null) return `${this.selector} ${this.comparatorKeyWord}`;
```
Comment (sisimomo) [inline src/comparators.ts]:
```suggestion
    if (typeof this.value === 'undefined' || this.value === null) {
      return `${this.selector} ${this.comparatorKeyWord}`;
    }
```

**Review by sisimomo — CHANGES_REQUESTED** [review]
I see, thanks for the fix. I’ll apply it once the code style is corrected.

**Comment by mok-liee** [comment]
done :)! big thanks

**Review by sisimomo — APPROVED** [review]

**Pull request merged**